client/components/footer-section/README.md

```diff
@@ -0,0 +1,18 @@
+# Footer Section (TSX)
+
+This component displays a footer section with a header and an space for content
+
+## How to use
+
+```js
+import FooterSection from 'calypso/components/footer-section';
+
+function render() {
+	return <FooterSection header={ header }>{ content }</FooterSection>;
+}
+```
+
+## Props
+
+- `header` (`ReactChild`) - The header string or a component to render as the header.
+- `children` (`ReactChild | ReactChild[]`) - The content to be rendered by the footer section.
```

Comment:
Is there a reason for specifying the where this is used (eg `Footer`). As I understand it can be used anywhere in Calypso pages. In which case, can we omit the `Footer` reference from all places in that component?

Reply:
Makes sense! will transform it into `Section` or something similar!

Reply:
✅